server/src/main/java/org/elasticsearch/cluster/ClusterStateTaskExecutor.java

```diff
@@ -24,24 +29,25 @@
      * surprisingly many tasks to process in the batch. If it's possible to accumulate the effects of the tasks at a lower level then you
      * should do that instead.
      *
+     * @param currentState The initial cluster state on which the tasks should be executed.
      * @param taskContexts A {@link TaskContext} for each task in the batch. Implementations must complete every context in the list.
+     * @return The resulting cluster state after executing all the tasks. If {code currentState} is returned then no update is published.
      */
     ClusterState execute(ClusterState currentState, List<TaskContext<T>> taskContexts) throws Exception;
 
     /**
-     * indicates whether this executor should only run if the current node is master
+     * @return {@code true} iff this executor should only run on the elected master.
```

Comment:
```suggestion
     * @return {@code true} if this executor should only run on the elected master.
```

Reply:
`iff` is what I intended here, as shorthand for `if and only if` https://en.wikipedia.org/wiki/If_and_only_if - we could spell that out if you'd prefer, but there are 176 other instances of this abbreviation in the codebase.